src/main/scala/de/uhh/lt/conll/CoNLLParser.scala

```diff
@@ -0,0 +1,69 @@
+package de.uhh.lt.conll
```

Comment:
putting conll outside jst is not consistent with other non-spark utilitites eg https://github.com/fmarten/JoSimText/blob/master/src/main/scala/de/uhh/lt/jst/utils/Util.scala which live INSIDE the jst. I would put conll inside the jst as well for this reason